[IMP] POS: update Products page (#18186)
odoo/documentation: content/applications/sales/point_of_sale/products.rst

========
Products
========

Odoo Point of Sale allows you to :ref:`create products <pos/products/creation>` and :ref:`manage
product information <pos/products/information-display>` in the POS register. Products can be
:ref:`configured <pos/products/configuration>` in various ways, including :ref:`variants
<pos/products/variants>`, :ref:`tags <pos/products/tags>`, :ref:`units <pos/products/units>`, and
:ref:`images <pos/products/images>`. You can also create :ref:`POS product categories
<pos/products/categories>`, use :ref:`product combos <pos/products/combos>`, and :ref:`manage

> the stock <pos/products/sn>`.

Reviewer: I'm a little confused because you removed mention of being able to create products from the backend, yet that's still the main focus of the 'Product creation' section, with creation from the POS resgister as a tip. Or maybe I'm missing something about your aim here?

I also think it might be good to rerrange the order of the topics mentioned, as I don't think the current phrasing works for e.g., units  ... two suggestions here:

```suggestion
Odoo Point of Sale allows you to :ref:`create products <pos/products/creation>` and :ref:`manage
product information <pos/products/information-display>` in the POS register or from the backend.
Products can be :ref:`configured <pos/products/configuration>` in various ways, including creating
:ref:`variants <pos/products/variants>`, :ref:`product combinations <pos/products/combos>`, and
:ref:`POS product categories <pos/products/categories>`, and adding :ref:`tags <pos/products/tags>`
and :ref:`images <pos/products/images>`. You can also group products by :ref:`units
<pos/products/units>` and :ref:`manage stock <pos/products/sn>`.
```
OR

```suggestion
Odoo Point of Sale allows you to :ref:`create products <pos/products/creation>` and :ref:`manage
product information <pos/products/information-display>` in the POS register or from the backend.
Products can be :ref:`configured <pos/products/configuration>` in various ways, including creating
:ref:`variants <pos/products/variants>` and :ref:`POS-specific product categories
<pos/products/categories>`, and adding :ref:`tags <pos/products/tags>` and :ref:`images
<pos/products/images>`. You can also create :ref:`product combinations <pos/products/combos>`, group
products by :ref:`units <pos/products/units>`, and :ref:`manage stock <pos/products/sn>`.
```

Author: I guess I removed it to make clear that "manage product information" is referring to the Product information display in the POS register, but now that makes no sense 😆 I will rephrase :) About the order: I wanted to follow the page's structure, but indeed, that is not necessary. I will consider your suggestions :) Thank you 🙌

.. _pos/products/creation:

> Product creation

Reviewer: Wouldn't it be better to structure this in a similar way to the eCommerce Products page, with first, how to create a product from the POS register, and then how to create from the backend?

Author: I did it this way here because there are these 7 advanced settings that you can't configure from the frontend. But we can discuss this if you want?

Reviewer: I definitely get that. But maybe that 'confirguration' information belongs in the configuration section?

<img width="970" height="313" alt="image" src="https://github.com/user-attachments/assets/7b593fc5-6e09-454f-af9e-c5f227ec9e98" />

Let's discuss!

  register <pos/products/information-display>` and in :doc:`self-ordering mode <extra/self_order>`
  when clicking the :icon:`fa-info-circle` :ref:`info button <pos/products/tag-application>`.
- In the :guilabel:`Upsell & cross-sell` section, enable :guilabel:`POS Optional Products` to
  suggest :doc:`optional products <../sales/sales_quotations/optional_products>` when :ref:`adding
  items to the cart <pos/use/sell>` in the POS register.
- If applicable, configure :doc:`Urpan Piper <restaurant/online_food_delivery>` and/or decide if the
  product is :guilabel:`Available for Platform Orders`, such as GrabFood or GoFood.

.. tip::
   It is also possible to create a product from the POS interface. To do so, open the :ref:`POS
   register <pos/use/open-register>`, click the :icon:`fa-bars` (:guilabel:`hamburger menu`) icon,
   then select :guilabel:`Create Product`. Enter the product details in the pop-up window and click
   :guilabel:`Save`. The product is immediately available in the POS register. Existing products can
   also be updated directly from the POS register through the :ref:`product information pop-up

>    window <pos/products/information-display>`.

Reviewer: I noticed that, if you don't provide a POS category, the product gets saved but is not visible in the POS register. Maybe we could/should add that to the doc (though if you create a product like this, ideally that field would be mandatory):

```suggestion
.. tip::
   It is also possible to create a product from the POS interface. To do so, open the :ref:`POS
   register <pos/use/open-register>`, click the :icon:`fa-bars` (:guilabel:`hamburger menu`) icon,
   then select :guilabel:`Create Product`. Enter the product details in the pop-up window, making sure to select
   a :ref:`POS product category <pos/products/categories>`, then click :guilabel:`Save`. The product is
   immediatele available in the POS register. Existing products can also be updated directly from the POS 
   register through the :ref:`product information pop-up window <pos/products/information-display>`.
```

Author: That is not entirely true, as you only have to add a category if you are using categories for the POS. I mean if you don't create any categories and you create the product from the POS register, it is added and displayed. It is just that you NEED a category if you are using at least 1 category. But you are right, I will rephrase a bit to make that even clearer

Reviewer: Aha, thanks for the explanation, I didn't realise that.

.. _pos/products/information-display:

Product information display in the POS register
-----------------------------------------------

To view and edit product information from the POS register, click and hold the product to access the
following details:

- the :doc:`available
  </applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/count_products>`
  and :doc:`forecasted
  </applications/inventory_and_mrp/inventory/warehouses_storage/reporting/forecast>` stock,
  including :doc:`warehouse
  </applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouses>`
  information,
- :doc:`replenishment periods
  </applications/inventory_and_mrp/inventory/warehouses_storage/replenishment>`,
- :doc:`optional products <../sales/sales_quotations/optional_products>`,
- financial information, such as :doc:`prices <extra/pricing>`, :ref:`taxes <pos/pricing/taxes>`,
  costs, and :doc:`margins <../sales/sales_quotations/margin>`.

You can also access and :guilabel:`Edit` product information after :ref:`adding a product to the
cart <pos/use/sell>` by clicking the :icon:`fa-ellipsis-v` menu, then :icon:`fa-info`

> :guilabel:`Info` button.

Reviewer: ```suggestion
You can also access and edit product information of a product that has already been :ref:`added to the cart <pos/use/sell>` by selecting the product, clicking the :icon:`fa-ellipsis-v` :guilabel:`(vertical ellipsis) icon, then the :icon:`fa-info` :guilabel:`Info` button.
```

- Using guilabel formatting for 'Edit' felt a bit premature
- If you have lots of items in the cart, then selecting the right one would be an important step, I guess

.. image:: products/product-information-window.png
   :alt: Example of product information window.
   :scale: 70%

.. tip::
   To display margins and costs in the product information window, go to the :ref:`POS settings
   <pos/use/settings>`, navigate to the :guilabel:`Product & PoS categories` section, and enable
   :guilabel:`Show margins & costs`.

.. _pos/products/configuration:

Product configuration
=====================

You can configure additional options, such as :ref:`product variants <pos/products/variants>`,
:ref:`product tags <pos/products/tags>`, :ref:`product and category images display
<pos/products/images>`, and :ref:`units of measure <pos/products/units>`.

.. _pos/products/variants:

Product variants
----------------

To use different :doc:`product variants
</applications/sales/sales/products_prices/products/variants>`, go to :menuselection:`Inventory -->
Configuration --> Settings`, scroll down to the :guilabel:`Products`, enable :guilabel:`Variants`,

> and :guilabel:`Save`.

Reviewer: It's a little odd here since, on the linked Product variants page (within the Sales app doc), we mention how to enable variants from the Sales app ... but of course, they might not have the Sales app, but will indeed have the Inventory app. Do you think there's any way to avoid this slight confusion? (Like to mention both app options. See my next comment for more on this ...) Or maybe you don't feel it's odd? 

Also, maybe:

```suggestion
To allow the use of :doc:`product variants
</applications/sales/sales/products_prices/products/variants>`, go to :menuselection:`Inventory -->
Configuration --> Settings`, scroll down to the :guilabel:`Products`, enable :guilabel:`Variants`,
and :guilabel:`Save`.
```

Author: I know what you mean, but I thought this was the only solution, but IMO it is not that odd. We are explaining how to enable Variants, but unfortunately, all the details are in the Sales documentation. But we could just remove the link to the Sales here so that the reader doesn't think of the Sales app and just follows the instructions for feature activation. Once activated, they can go to the detailed Product Variant page in the Sales documentation and find the configuration information they need. I guess at this point, they don't care anymore that you could also enable it from the Sales app (that they have probably not installed). Do you understand what I mean?

Reviewer: Yeah, maybe it's better to remove the link in this sentence (in any case, you linked to the sales doc in the previous paragraph), so they just go ahead and follow the inx to activate it via Inventory.

.. note::

>    The **Inventory app** is automatically installed when using the **Point of Sale app**.

Reviewer: ```suggestion
   The **Inventory** app is automatically installed when using the **Point of Sale** app.
```

Then navigate to :menuselection:`Point of Sale --> Products --> Products`, select a product, and
configure :ref:`attributes <products/variants/attributes>` and their :ref:`values
<products/variants/attributes-values>` to create :ref:`variants <products/variants/variants>` in
the :guilabel:`Attributes & Variants` tab. On the value's form, enable :guilabel:`Available on Food

> Delivery` for values available through delivery platforms.

Reviewer: It's a shame that on the Sales app pages we're linking to, it only mentions Sales app > Configuration > Attributes / Sales app > Product > Product ... do you think that could be a bit confusing or off-putting? Do you think it would work (or be worth it) to update those sections to mention something like:

- **Sales app** > **Configuration** > **Attributes** (or **Point of Sale** > **Configuration** > **Attributes or Website** > **eCommerce > Attributes**)

- **Sales app** > **Product** > **Product** (or **Point of Sale** > **Product** > **Product** or **Website** > **eCommerce** > **Product**)

Author: See my previous comment.

Reviewer: On re-reading, I think it's ok like this. You tell them to go to Point of Sale ‣ Products ‣ Products, so they should be able to figure it out.

> Delivery` for values available through delivery platforms.

Reviewer: Should the point about the 'Available on food delivery' be a tip maybe?

Author: This is the only POS-specific setting for variants, so I thought to leave it here, but we could add an "if applicable" to emphasize that this is not there by default

.. tip::
   Navigate to :menuselection:`Point of Sale --> Configuration --> Attributes` to manage all
   configured attributes, or to :menuselection:`Point of Sale --> Products --> Product Variants` to
   view all variants.

.. admonition:: Practical application

   Users in the :ref:`POS register <pos/use/open-register>` and customers in :doc:`self-ordering
   mode <extra/self_order>` can select product variants when :ref:`adding items to the cart
   <pos/use/sell>`.

     .. image:: products/product_variants_kiosk.png
        :alt: Example of product variants in self-ordering mode.
        :scale: 60%

.. _pos/products/tags:

Product Tags
------------

> Product tags help categorize products and highlight specific features. To create product tags, go to

Reviewer: Since tags are only visible in self-ordering mode, I think it's worth mentioning that when we first introduce them, no? Still nice to show the practical application, of course.

:menuselection:`Point of Sale --> Configuration --> Product Tags`. On the :guilabel:`Product Tags`
page, click :guilabel:`New` or select an existing tag.

On the product tag form:

- Enter a :guilabel:`Tag` label.

> - Enable/disable :guilabel:`Visible to customers` to control whether it is visible to customers in

Reviewer: ```suggestion
- Enable/disable :guilabel:`Visible to customers` to control whether the tag is visible to customers in
```

  :doc:`self-ordering mode <extra/self_order>`.
- Choose a :guilabel:`Color` if needed.

> - Add an image if needed. The image replaces the tag's color if both are configured.

Reviewer: Suggest to combine these, especially since only one can be used:

```suggestion
- Optionally, choose a :guilabel:`Color` or add an image. If both are configured, the image replaces the tag's color. 
```

- In the :guilabel:`Point of sale` section, write a description to be shown in self-ordering mode.

To assign it to a product, navigate to the product form, go to the :guilabel:`Sales` tab, and, in
the :guilabel:`Extra info` section, add as many :guilabel:`Tags` as needed.

.. note::
   If the eCommerce app is installed, tags are used in both apps. Therefore, they must be added in

>    the :guilabel:`Ecommerce shop` section of the :guilabel:`Sales` tab.

Reviewer: First change suggested so we are using the same wording in the note:

```suggestion
To add tags to a product, navigate to the product form, go to the :guilabel:`Sales` tab, and, in
the :guilabel:`Extra info` section, add as many :guilabel:`Tags` as needed.

.. note::
   If the **eCommerce** app is installed, tags are used in both apps, and need to be added in the :guilabel:`Ecommerce shop` section of the :guilabel:`Sales` tab.
```

.. _pos/products/tag-application:

.. admonition:: Practical application

   Tags are displayed only in :doc:`self-ordering mode <extra/self_order>`, *not* in the :ref:`POS
   register <pos/use/open-register>`. Customers can view the tags and tag descriptions by clicking

>    the :icon:`fa-info` info icon next to the product name.

Reviewer: ```suggestion
   register <pos/use/open-register>`. Customers can view the tags and tag descriptions by clicking
   the :icon:`fa-info` :guilabel:`(info)` icon next to the product name.
```

   .. image:: products/tag_kiosk.png
      :alt: Example of product tags in self-ordering mode.
      :scale: 60%

.. _pos/products/units:

Product units
-------------

Products can be grouped by :doc:`unit of measure
</applications/inventory_and_mrp/inventory/product_management/configure/uom>` in the :ref:`POS
register <pos/use/open-register>` and on the :doc:`customer display
<hardware_network/customer_display>`.

To do so,

#. Enable :ref:`Units of Measure & Packagings <inventory/product_replenishment/configuration>`.
#. Use the :icon:`fa-arrow-right` :guilabel:`Units & Packagings` link to view all configured units.
#. Enable :ref:`developer-mode`.
#. Select a unit form.

> #. Enable :guilabel:`Group Products in POS`.

Reviewer: A few suggestions:

```suggestion
To do so, follow these steps:

#. Enable :ref:`Units of Measure & Packagings <inventory/product_replenishment/configuration>` in the Inventory app settings.
#. Click the :icon:`fa-arrow-right` :guilabel:`Units & Packagings` link to view all configured units.
#. Select a unit.
#. Activate :ref:`developer mode<developer-mode>`.
#. Enable :guilabel:`Group Products in POS`.
```

Happy to discuss my thoughts on any of these!

the :ref:`POS register <pos/use/open-register>`, on the :doc:`customer display
<hardware_network/customer_display>`, and/or in :doc:`self-ordering mode <extra/self_order>`.

Once you have added images to :ref:`products <pos/products/creation>` or :ref:`categories
<pos/products/categories>`, configure their visibility. To do so, navigate to the :ref:`POS settings
<pos/use/settings>`, scroll down to the :guilabel:`POS interface` section, and under :guilabel:`Hide
pictures in POS`, enable/disable :guilabel:`Show product images` and/or :guilabel:`Show category
images`.

.. note::
   These settings only affect the POS register, *not* the self-ordering mode or the customer
   display.

.. admonition:: Practical application

   - **Product images** can be displayed in the POS register, on the customer display, and on the
     :guilabel:`Your Order` page in self-ordering mode.
   - **Category images** can be displayed in the POS register.
   - **Product variant images** are displayed on the customer display and on the :guilabel:`Your
     Order` page in self-ordering mode.
   - **Product tag images** are displayed when :ref:`clicking the info button

>      <pos/products/tag-application>` in self-ordering mode.

Reviewer: I feel we could make this more concise. How about something like:

```suggestion
Product and category images
---------------------------

Images can be added for products, :ref:`product tags <pos/products/tags>`, :ref:`variants
<pos/products/variants>`, and :ref:`categories <pos/products/categories>`. They can be displayed in
the :ref:`POS register <pos/use/open-register>`, on the :doc:`customer display
<hardware_network/customer_display>`, and/or in :doc:`self-ordering mode <extra/self_order>`.

Once you have added images to :ref:`products <pos/products/creation>` or :ref:`categories
<pos/products/categories>`, configure their visibility. To do so, navigate to the :ref:`POS settings
<pos/use/settings>`, scroll down to the :guilabel:`POS interface` section, and under :guilabel:`Hide
pictures in POS`, enable/disable :guilabel:`Show product images` and/or :guilabel:`Show category
images`.

.. note::
   These settings only affect the POS register, *not* the self-ordering mode or the customer
   display.

.. admonition:: Practical application

   - **Product images** can be displayed in the POS register, on the customer display, and on the
     :guilabel:`Your Order` page in self-ordering mode.
   - **Category images** can be displayed in the POS register.
   - **Product variant images** are displayed on the customer display and on the :guilabel:`Your
     Order` page in self-ordering mode.
   - **Product tag images** are displayed when :ref:`clicking the info button
     <pos/products/tag-application>` in self-ordering mode.
     hgfhggf
```

Author: @mcsi-odoo I think something went wrong here. I cannot see your suggestion. Or am I missing something?

Reviewer: Hmm, not sure what happened!! And now of course I don't remember what I had suggested ... let me try to dig it from my memory.

>      <pos/products/tag-application>` in self-ordering mode.

Reviewer: So my idea was something like this:

```suggestion
Images can be added for :ref:`products <pos/products/creation>`, :ref:`product tags
<pos/products/tags>`, :ref:`variants <pos/products/variants>`, and :ref:`categories
<pos/products/categories>`. They can be displayed in the :ref:`POS register
<pos/use/open-register>`, on the :doc:`customer display <hardware_network/customer_display>`, and/or
in :doc:`self-ordering mode <extra/self_order>`, as follows:

.. list-table::
   :header-rows: 1
   :stub-columns: 1
   :widths: 5 5 5 5

   * - 
     - POS register
     - Customer display
     - Self-ordering mode
   * - Product image
     - :icon:`fa-check`
     - :icon:`fa-check`
     - :icon:`fa-check` on the :guilabel:`Your Order` page
   * - Product tag image
     - :icon:`oi-close`
     - :icon:`oi-close`
     - :icon:`fa-check` when clicking the :icon:`fa-info-circle` :ref:`Info button
       <pos/products/tag-application>`
   * - Product variant image
     - :icon:`oi-close`
     - :icon:`fa-check`
     - :icon:`fa-check` on the :guilabel:`Your Order` page
   * - Category image
     - :icon:`fa-check`
     - :icon:`oi-close`
     - :icon:`oi-close`

Once you have added images to products or categories, configure their visibility *in the POS
register*. To do so, navigate to the :ref:`POS settings <pos/use/settings>`, scroll down to the
:guilabel:`POS interface` section, and under :guilabel:`Hide pictures in POS`, enable/disable
:guilabel:`Show product images` and/or :guilabel:`Show category images`.
```

.. _pos/products/categories:

POS product categories
======================

POS product categories are used to organize products in the POS register. Products are grouped by

> categories to enhance navigation.

Reviewer: Suggestion to avoid repetition:

```suggestion
POS product categories are used to group products in the POS register to aid navigation.
```

To manage POS categories, follow these steps:

#. Navigate to :menuselection:`Point of Sale --> Configuration --> PoS Product Categories`.
#. Click :guilabel:`New` to create a category or select an existing one to update it.
#. On the category form:

   - Enter a label for the category.
   - Build a hierarchy of categories: Associate a category with a parent
     category by filling in the :guilabel:`Parent Category` field. A parent category groups one or

>      more child categories (e.g., use `Drinks` to group `Hot beverages` and `Soft drinks`).

Reviewer: ```suggestion
   - Enter or edit the label for the category.
   - Select a :guilabel:`Parent Category` to build a hierarchy of categories. A parent category groups one or more child categories (e.g., use `Drinks` to group `Hot beverages` and `Soft drinks`).
```

   - Define a :guilabel:`Color` for display in the POS register.
   - Add an image if needed.
   - If :doc:`Online food delivery platforms
     </applications/sales/point_of_sale/restaurant/online_food_delivery>` are configured in your
     database, indicate their :guilabel:`Service hours`.

After creating POS product categories, assign them to specific products on the :ref:`product form
<pos/products/creation>`.

To limit categories displayed in the POS register, go to the :ref:`POS settings <pos/use/settings>`,
navigate to the :guilabel:`Product & PoS categories` section, and select the relevant categories in
the :guilabel:`Restrict Categories` field.

.. admonition:: Practical application

   Product categories are displayed in the POS register and self-ordering mode. Subcategories appear

>    after selecting a parent category.

Reviewer: ```suggestion
   Product categories are displayed in the :ref:`POS register <pos/use/open-register>` and in :doc:`self-ordering
   mode <extra/self_order>`. Subcategories appear after selecting a parent category.
```

   .. image:: products/categories-in-kiosk.png
      :alt: Categories and subcategories in kiosk mode.

.. tip::
   If no category is selected in the POS register, all products are displayed. To organize products
   by category in this view, navigate to the :ref:`POS settings <pos/use/settings>`, scroll down to
   the :guilabel:`PoS Interface` section, and enable :guilabel:`Group products by categories`.

   .. example::
      In this POS register, two categories are configured, :guilabel:`Food` and :guilabel:`Drinks`,
      but neither of them is currently selected. The overview shows all products in both categories,

>       grouped by category on separate rows.

Reviewer: In the example, are the different colors due to subcategories?
To make the example more obvious, and avoid any questions, it might be clearer if the colors matched the categories if you see what I mean? i.e., if all the food has a pink edge, and all the drinks green

_Edited to add_
OK, I guess you can just add any of the colors to any product. If you think it's fine like this, then leave it. (I was confused because when I was trying with a clothes shop in a runbot, each category was 'nicely' a different color.

Author: I will try to add a new screenshot that makes it clearer. You are right :)

      .. image:: products/group-by-category.png
         :alt: Group products by categories.

.. _pos/products/combos:

Product combos
==============

A product combo is a bundle of multiple products sold together as a single unit. Each product combo
consists of multiple categories, known as :ref:`combo choices <pos/products/combo-choices>`, and
each combo choice contains several items. When purchasing a product combo, customers can select one
or more items from each combo choice.

.. example::
    A burger menu is offered as a product combo including three combo choices: one burger, one
    drink, and one portion of fries. For each combo choice, customers select one item from the
    available options (e.g., cheese burger or bacon burger; water or soda; large or regular fries).

    .. image:: products/combo-selection-in-pos.png
       :alt: Combo selection in the POS register.
       :scale: 70%

    You can still change your selection after :ref:`adding it to the cart <pos/use/sell>`. To

>     do so, click and hold the combo item in the cart to open the selection pop-up window.

Reviewer: Not sure about the 'You can ...'. And maybe add it as a tip?

```suggestion
    .. tip::
       To change the selection after :ref:`adding a product combo to the cart <pos/use/sell>`, click and hold the
       combo item in the cart to re-open the selection pop-up window.
```

Products can be :ref:`configured <pos/products/configuration>` in various ways, including creating
:ref:`variants <pos/products/variants>`, :ref:`product combinations <pos/products/combos>`, and
:ref:`POS-specific product categories <pos/products/categories>`, and adding :ref:`tags
<pos/products/tags>` and :ref:`images <pos/products/images>`. You can also group products by
:ref:`units <pos/products/units>` and :ref:`manage stock <pos/products/sn>`.

.. _pos/products/creation:

Product creation
================

To create products from the backend, go to :menuselection:`Point of Sale --> Products --> Products`,
and click :guilabel:`New` to create a product, or select an existing one to edit it.
Ensure the :guilabel:`Point of Sale` checkbox is enabled at the top of the form, then
:doc:`configure the product's general information
</applications/inventory_and_mrp/inventory/product_management/configure>`, such as defining the
:doc:`product type </applications/inventory_and_mrp/inventory/product_management/configure/type>`,
setting a :ref:`tracking strategy <pos/products/sn>`, or adding a product image, as needed.

> 

Reviewer: Maybe you still need to create a link to the Product configuration section with something like:

```suggestion
Configure advanced, POS-specific :ref:`product details <pos/products/configuration>` as needed.
```

.. tip::
   It is also possible to create a product from the POS interface. To do so, open the :ref:`POS
   register <pos/use/open-register>`, click the :icon:`fa-bars` (:guilabel:`hamburger menu`) icon,
   then select :guilabel:`Create Product`. Enter the product details in the popover and click
   :guilabel:`Save`. The product is immediately available in the POS register. Existing products can
   also be updated directly from the POS register through the :ref:`product information popover
   <pos/products/information-display>`.

   .. note::
      When using :ref:`POS product categories <pos/products/categories>`, each product must be

>       assigned to a category. Otherwise, it will not be displayed or available for selection.

Reviewer: ```suggestion
      When using :ref:`POS product categories <pos/products/categories>` each product must be
      assigned to a :guilabel:`POS Category`, otherwise the product will not be displayed or available for selection.
```

Or just "... will not appear in the POS register"?

.. seealso::
   `Product creation (video tutorial) <https://youtu.be/b5eVusXHEvg?si=Xn3EBMmRfJ35mqyu>`_

.. _pos/products/information-display:

Product information display in the POS register
-----------------------------------------------

To view and edit product information from the POS register, click and hold the product to access the
following details:

- the :doc:`available
  </applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/count_products>`
  and :doc:`forecasted
  </applications/inventory_and_mrp/inventory/warehouses_storage/reporting/forecast>` stock,
  including :doc:`warehouse
  </applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouses>`
  information,
- :doc:`replenishment periods
  </applications/inventory_and_mrp/inventory/warehouses_storage/replenishment>`,
- :doc:`optional products <../sales/sales_quotations/optional_products>`,
- financial information, such as :doc:`prices <extra/pricing>`, :ref:`taxes <pos/pricing/taxes>`,
  costs, and :doc:`margins <../sales/sales_quotations/margin>`.

> You can also access and edit product information of a product that has already been :ref:`added to

Reviewer: ```suggestion
You can also access and edit the product information of a product that has already been :ref:`added to
```

register*. To do so, navigate to the :ref:`POS settings <pos/use/settings>`, scroll down to the
:guilabel:`POS interface` section, and under :guilabel:`Hide pictures in POS`, enable/disable
:guilabel:`Show product images` and/or :guilabel:`Show category images`.

.. _pos/products/categories:

POS product categories
======================

POS product categories are used to group products in the POS register to aid navigation.

To manage POS categories, follow these steps:

#. Navigate to :menuselection:`Point of Sale --> Configuration --> PoS Product Categories`.
#. Click :guilabel:`New` to create a category or select an existing one to update it.
#. On the category form:

   - Enter or edit the label for the category.
   - Select a :guilabel:`Parent Category` to build a hierarchy of categories. A parent category
     groups one or more child categories (e.g., use `Drinks` to group `Hot beverages` and `Soft
     drinks`).
   - Define a :guilabel:`Color` for display in the POS register.
   - Add an image if needed.
   - If :doc:`Online food delivery platforms
     </applications/sales/point_of_sale/restaurant/online_food_delivery>` are configured in your

>      database, indicate their :guilabel:`Service hours`.

Reviewer: ```suggestion
   - If :doc:`online food delivery platforms
     </applications/sales/point_of_sale/restaurant/online_food_delivery>` are configured in your
     database, indicate their :guilabel:`Service Hours`.
```

After creating POS product categories, assign them to specific products on the :ref:`product form
<pos/products/creation>`.

To limit categories displayed in the POS register, go to the :ref:`POS settings <pos/use/settings>`,
navigate to the :guilabel:`Product & PoS categories` section, and select the relevant categories in

> the :guilabel:`Restrict Categories` field.

Reviewer: Since the wording on the UI is kinda iffy, maybe we can be more helpful here:

<img width="397" height="151" alt="Image" src="https://github.com/user-attachments/assets/261c95d4-0b2f-40e6-a2c0-b4eb47fb849d" />

```suggestion
To only display specific categories in the POS register, go to the :ref:`POS settings <pos/use/settings>`,
navigate to the :guilabel:`Product & PoS categories` section, then, in the :guilabel:`Restrict Categories` field, select the categories that should be available.
```

#. Optionally, click the :guilabel:`Point of Sale` tab to select the preferred :guilabel:`Category`.

.. note::
   The total price of the product combo displayed in the :ref:`POS register
   <pos/use/open-register>` is based on the :guilabel:`Sales Price` defined on the :ref:`product
   combo's form <pos/products/combo-creation>`. Selecting several products in the combo choices and
   selecting a product with an :guilabel:`Extra Price` influences the total price.

   .. example::
      The :guilabel:`Office Combo` has a :guilabel:`Sales Price` of **300** € and offers a
      selection of chairs and desks. The combo choice for chairs includes a conference chair, an
      office chair, and an armchair, with a maximum selectable amount set to 2. The :guilabel:`Combo
      Price` for the chair combo choice is **35** € per added item, since the conference chair is
      the least expensive product. The armchair has an :guilabel:`Extra Price` of **100** € because
      it is made of leather. Selecting both the conference chair and the armchair increases the
      price of the :guilabel:`Office Combo` to **435** €. Thus, selecting the armchair adds **35** €

>       (:guilabel:`Combo Price`) + **100** € (:guilabel:`Extra Price`).

Reviewer: I struggled with this a bit ... could some further rewording make it easier to understand?

```suggestion
   .. example::
      The :guilabel:`Office Combo` has a :guilabel:`Sales Price` of **300** € and offers a
      selection of chairs and desks. The combo choice for chairs includes a conference chair, an
      office chair, and an armchair, with a maximum selectable amount set to 2. For each
      additional chair a customer selects, a :guilabel:`Combo Price` of **35** € is added. The
      armchair has an :guilabel:`Extra Price` of **100** € because it is made of leather. Selecting
      both the conference chair and the armchair increases the price of the :guilabel:`Office
      Combo` to **435** €, i.e., :guilabel:`Sales Price` + **35** € (:guilabel:`Combo Price`) for
      adding a second chair + **100** € (:guilabel:`Extra Price`) for the leather armchair.
```
 
I don't think the 'because the conference chair is the least expensive product' is helpful here, since we're not mentioning its price anyway.

      .. image:: products/office-combo.png
         :scale: 60%

.. _pos/products/stock-management:

Stock management
================

To track stock for POS products, the :ref:`product type must be Goods or Combo

> <inventory/product_management/goods-or-services>`, and the :ref:`track inventory option

Reviewer: ```suggestion
To track stock for POS products, the :ref:`product type <inventory/product_management/goods-or-services>` must be set to :guilabel:`Goods` or :guilabel:`Combo`, and the :ref:`track inventory option
```

<inventory/product_management/tracking-inventory>` must be enabled. Products can be tracked by

> quantity, :ref:`by lot, or by unique serial number <pos/products/sn>`.

Reviewer: ```suggestion
quantity, :ref:`by lot, or by serial number <pos/products/sn>`.
```

Unless the 'unique' is there for a specific reason?

Author: I added it because that is how it is named when selecting the tracking option on the product form. But we can also remove it

<img width="208" height="110" alt="Image" src="https://github.com/user-attachments/assets/589b1d4d-63da-414c-aa03-3a75dc0c7c98" />

.. tip::
   When adding tracked items to the :ref:`cart <pos/use/sell>`, the number of :guilabel:`Free To
   Use` units is displayed in the POS configuration pop-up window.

   .. image:: products/pos-available-stock.png
      :alt: Free-to-use stock displayed in POS register.
      :scale: 70%

.. _pos/products/sn:

Serial numbers and lots
-----------------------

> Using **lots** and **serial numbers** allows for tracking product movements throughout their

Reviewer: I think my dislike of this formulation ('allows for [gerund]') may be a very personal thing, but I'll present some options nonetheless 😉 :

```suggestion
Using **lots** and **serial numbers** allows for the tracking of product movements throughout a product's lifecycle. When traceability is enabled, Odoo identifies a product's location based on its last
```

```suggestion
Using **lots** and **serial numbers** allows product movements to be tracked throughout a product's lifecycle. When traceability is enabled, Odoo identifies a product's location based on its last
```

lifecycle. When traceability is enabled, Odoo identifies a product's location based on its last
recorded movement.

To track products by lots or serial numbers:

#. :ref:`Enable the Lots & Serial Numbers setting
   <inventory/product_management/traceability-setting>`.
#. :ref:`Configure products <pos/products/creation>`.
#. :ref:`Assign tracking numbers <inventory/product_management/assign-sn>`.

.. tip::
   :doc:`Expiration dates
   </applications/inventory_and_mrp/inventory/product_management/product_tracking/expiration_dates>`
   can also be configured for products tracked by lot or serial number.

.. _pos/products/tracking:

Selling tracked products
~~~~~~~~~~~~~~~~~~~~~~~~

Adding a tracked product to the cart from the POS register automatically imports its serial number
or lot number. If no tracking number exists, a pop-up window appears to create and assign one.

.. image:: products/create-serial-number.png
   :alt: Popup to create and assign a serial number.

.. note::
   When you :ref:`load a quotation/SO <pos/shop/so>` containing tracked products, a pop-up window
   asks whether the numbers linked to the :abbr:`SO (sales order)` should be imported. Click

>    :guilabel:`Ok` to proceed.

Reviewer: ```suggestion
   When you :ref:`import a quotation or sales order <pos/shop/so>` containing tracked products, a pop-up window
      asks whether the numbers linked to the quotation or sales order should be imported. Click
   :guilabel:`Ok` to proceed.
```

- We use 'import' on the linked doc page, so maybe that's better than 'load'?
- I'm not sure about using the abbreviation ... especially if it's both quotation and sales order ... though does it ever make sense to be importing a quotation?

Author: I kept the "load an order" phrasing because there is a "Load order" button in the POS register (when you click the Orders tab and select an order) but we don#t have to use "load" :)

<img width="1920" height="993" alt="Image" src="https://github.com/user-attachments/assets/a75bf1b4-3a27-4a99-aef6-07f7db5939e0" />

Reviewer: Actually, maybe load is better then ... what would you think about updating this to 'loaded':

<img width="742" height="351" alt="image" src="https://github.com/user-attachments/assets/b74af502-0f4b-4cf2-8499-df27daafaec3" />

Once tracking numbers are assigned or imported, they appear in the cart below the corresponding
products, next to the :icon:`fa-list` (:guilabel:`Valid product lot`) icon. The icon's color
highlights if the tracking has been assigned:

- **Green** :icon:`fa-list` (:guilabel:`Valid product lot`) **icon**: The tracking number was
  successfully imported or assigned.
- **Red** :icon:`fa-list` (:guilabel:`Invalid product lot`) **icon**: The tracking number is
  missing or incorrect.

To change a tracking number, click the :icon:`fa-list` (:guilabel:`Valid product lot`) icon, select
a different lot or serial number from the pop-up window, and click :guilabel:`Apply`.

.. example::
   The icon shows green for the :guilabel:`Casual T-shirt` as the serial number `123456` has been
   assigned. On the other hand, the icon appears in red for the :guilabel:`Cozy sweater` because no

>    serial number has been assigned.

Reviewer: ```suggestion
   The icon appears in green for the :guilabel:`Casual T-shirt` as the serial number `123456` has been
   assigned. On the other hand, the icon is red for the :guilabel:`Cozy sweater` because no
   serial number has been assigned.
```
